config/application.rb

```diff
@@ -176,7 +176,8 @@ class Application < Rails::Application
       :ldap => {:enabled => false},
       :permissions => {:enabled => false},
       :sql => {:enabled => false},
-      :templates => {:enabled => true}
+      :templates => {:enabled => true},
+      :audit => {:enabled => true}
```

Comment:
This list's alphabetical, it's out of order.

(I'm also unsure why this is called "audit" when the fact import instrumentation isn't related to audits, seems like a poor name.)